db/migrations/campaign/V4.0.0.4__table_campaigns.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE IF NOT EXISTS campaign.beneficiaries (
+    id uuid NOT NULL DEFAULT (gen_random_uuid()),
+    name character varying(100) NOT NULL,
+    type integer NOT NULL,
+    iso2country_code integer NOT NULL,
+    city character varying(50) NOT NULL,
+    email character varying(100) NULL,
+    phone character varying(50) NULL,
+    website character varying(500) NULL,
```

Comment:
I am wondering, maybe better store these additional properties like email, phone, website in a jsonb column? What do you think @kachar @igoychev ?

Reply:
I think `email` is quite important to be present as indexable column with strict type check.

It's going to be nice to have additional props as jsonb column as you say to collect metadata about the user that requires no special db structure as we can't properly type the elements of a json field.

An example might be the `website` field which may have a type check for `http://` or `https://` which will guarantee db integrity.

Reply:
Of course these ideas above are not mandatory for this iteration, we can implement them in the next sprints as well.